functions/exponents_and_logs.py

```diff
@@ -146,7 +146,7 @@ def pow(base: float, exponent: float, root_used: int = int(1E10)) -> float:
     fractional_part_of_exponent = exponent - integer_part_of_exponent
     # Ensure that base is not negative if exponent is not a integer.
     if common.is_negative(base) and fractional_part_of_exponent != 0:
-        raise exceptions.InputError(None, "Implementation of exponent function for real exponents does not support negative bases because the power function for negative bases is non-continuous over the set of real numbers.")
+        raise exceptions.InputError(None, "negative base with fractional exponent.")
```

Comment:
Watch out for capitalization for error message, should probably have them all be similar